haskell/yesod/README.md

```diff
@@ -0,0 +1,67 @@
+# yesod docker image
+
+Use this image as the base image for your yesod based applications.
+
+## Usage
+
+The simplest usage is to create a `Dockerfile` for your project that looks like
+this:
+```
+FROM thoughtbot/yesod:1.4.10-2
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY *.cabal ./
+RUN cabal install --dependencies-only -j4
+```
+
+You can build your container like this:
```

Comment:
Do we want to build the container here? I think that will install all the dependencies, and it looks like you'll have to install all the dependencies again when running `docker-compose build`.